modules/wallet/persist.go

```diff
@@ -117,6 +137,59 @@ func (w *Wallet) CreateBackup(backupFilepath string) error {
 	return w.createBackup(f)
 }
 
+type compatPersist struct {
```

Comment:
Need a version number in all compat type names. It'll make life a lot easier down the road.